docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Apache Kafka® MirrorMaker 2 log analysis using the offset sync tool
+sidebar_label: Offset sync log analysis
+---
+
+Learn how to analyze key log messages generated during data synchronization with Apache Kafka® MirrorMaker 2.
+These logs are produced by the offset sync inspection tool, which you can run locally
```

Comment:
This needs a little more context. 

"Aiven has a tool to be able to track the progress when using MM2 between source and destination clusters. This is called the offset inspection tool, which you can run locally after setting up the required environment to review offset syncing between source and
target Apache Kafka topics.  The tool generates messages showing partition health, and whether intervention is needed. This may be useful, especially if you are migrating a large amount of data

Reply:
Updated the content